tests-legacy/check_file_syntax.sh

```diff
@@ -25,5 +25,8 @@ if [[ "$php" == "0" && "$yaml_src" == "0" && "$yaml_app" == "0" && "$yaml_themes
   exit 0;
 else
   echo -e "\e[91mSYNTAX TESTS FAILED"
+  if [[ "$coding_styles" -ne "0" ]]; then
+   echo -e "\e[91mRun \"php ./vendor/bin/php-cs-fixer fix\" from your PrestaShop folder to fix the lint issues.";
```

Comment:
`./vendor/bin/php-cs-fixer` is a shell script, not PHP. The correct command should be `./vendor/bin/php-cs-fixer fix`.

Note that it might be nice to inform the contributor they need to install it first via composer, e.g. `composer install`. Though it seems unlikely they'd not already have done so :).

Reply:
`./vendor/bin/php-cs-fixer` is a PHP script under Linux and began with the shebang `#!/usr/bin/env php`

Reply:
@PierreRambaud I invite you to try this under Windows

Reply:
But here's the thing: as it stands, with `php` explicitly mentioned, this does *not* work under Windows. It is superfluous--as you mentioned, the script already defines the environment--so it can safely be removed. I just checked this to be safe on Ubuntu, and indeed it still works fine after. Doing so, however, *also* makes it work under Windows. IIUC, there are no downsides here to dropping `php` and having the bot tell users to run `./vendor/bin/php-cs-fixer fix`. That's nicely cross-platform. Do you agree?

Reply:
Even if there is a shebang, nothing define this script is executable, we don't do any `chmod +x` before. And still because we are in sh script context and this one is not cross-platform, we can keep the `php` runner :)

Reply:
You should not if you're going to advise to run this to any contributor that makes a style error (after all, what do you know of their system?), but very well. I'll leave it to @Quetzacoalt91.

Reply:
For that we should have a bat script (or don't know what powershell used) instead. Shell are created for Unix system only.

Reply:
I fear you might be confused, for which I apologise. Allow me to explain better.

I'm not talking about `tests-legacy/check_file_syntax.sh`. That's executed on Travis - fine by me. I'm talking about the recommendation given: that the contributor should run `php ./vendor/bin/php-cs-fixer fix` on his/her local dev machine. This specific command does not work under Windows, whereas `./vendor/bin/php-cs-fixer fix` does. That still works under Linux, making it nicely cross-platform. As such, I think the recommendation should become `./vendor/bin/php-cs-fixer fix` - i.e., only the line I commented on should be changed.

Reply:
Hahaha yes I was not thinking about that :rofl: 
In this case you're right, we can remove the `php` executable :)
My bad for the misunderstanding ^^

Reply:
Great, I'm glad we cleared that up!

Reply:
This command does not work either because of the slashes. An actual command on the good old `cmd` is: `.\vendor\bin\php-cs-fixer fix`